src/librustdoc/html/static/styles/main.css

```diff
@@ -26,9 +26,6 @@ h1.fqn {
 h2, h3:not(.impl):not(.method):not(.type):not(.tymethod), h4:not(.method):not(.type):not(.tymethod) {
 	border-bottom-color: #DDDDDD;
 }
-.in-band {
-	background-color: white;
-}
```

Comment:
Hmm, that makes sense, but i'm still gonna call in @GuillaumeGomez for a second opinion. (Usually he's supposed to get pinged on CSS changes but it looks like that one didn't happen this time `>_>`)